pages/home/glossary.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: "Glossary"
+description: "Glossary of terms used in the Arcade platform"
+---
+
+# Arcade Glossary
+
+
+## Agents and Tools
+
+```mermaid
+graph TD
+    A[Agent] --> B[Toolkit]
+    B --> C[Tool]
+```
+
+### Agent
+
+An 'agent' is the application you are building.  It can be a chatbot, a web application, a mobile app, or any other type of application that happens to use an LLM to as part of its functionality.  Agents interact with the world by calling tools.  Helping you build, test, authenticate, and deploy tools is what Arcade is all about.
+
+### Tool
+
+A 'tool' is a function that can be called by an agent which interacts with another service - commonly via an API, filesystem, database etc.  Tools are written in python and deployed by running a worker which contains the toolkit's code.  Tools are defined by the `@tool()` decorator and will be passed `ToolContext` as the first argument.  If a tool has dependencies that are not met (a secret is not provided, for example), the tool will fail to execute.
+
+An example of a tool, scoped by it's toolkit would be `Gmail.SendEmail`.
+
+*Learn more about [tools](/home/build-tools/create-a-toolkit).*
+
+### Toolkit
```

Comment:
Kinda feels like we should define this before 'tool'. Agent->toolkit->tool

Reply:
91236394b1d4a9db19a70160745b417509c64b13